### PR TITLE
Refactor and remove DBUtils

### PR DESCRIPTION
**Pull request by AlexandraRoatis**
## Type of change

- [ ] Bug fix.
- [ ] New feature.
- [x] Enhancement.
- [ ] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

### CODE REVIEW

File: modAionImpl/src/org/aion/zero/impl/blockchain/AionBlockchainImpl.java [hunk 1]
```diff
@@ -2767,4 +2751,179 @@ public void pruneOrRecoverState(boolean dropArchive, AionGenesis genesis, Logger
...
+                                LOG.info("The repository state trie was:\n");
+                                LOG.info(repository.getTrieDump());
+                            }
+
+                            fail = true;
+                            break;
+                        }
+
+                        if (currentBlock % stepSize == 0) {
+                            double time = System.currentTimeMillis() - start;
```
Comment (AionJayT) [inline modAionImpl/src/org/aion/zero/impl/blockchain/AionBlockchainImpl.java]: I prefer the calculation to use long directly. Even it has a little rounding issue.
But we can refactor it in a new task.

The estimate can be represent by TimeUnit conversion.

File: modAionImpl/src/org/aion/zero/impl/cli/DevCLI.java [hunk 2]
```diff
@@ -61,24 +62,39 @@ public static void printUsage(PrintStream stream, DevCLI instance) {
     }
 
     public static Cli.ReturnType printAccountDetails(String strAddress) {
+        // TODO: more parameters would be useful, e.g. get account X at block Y
```
Comment (AionJayT) [inline modAionImpl/src/org/aion/zero/impl/cli/DevCLI.java]: Convert TODO to a Jira ticket.

Comment (AlexandraRoatis) [reply modAionImpl/src/org/aion/zero/impl/cli/DevCLI.java]: done

File: modAionImpl/src/org/aion/zero/impl/blockchain/AionBlockchainImpl.java (hunk 1 above)
Comment (AlexandraRoatis) [reply modAionImpl/src/org/aion/zero/impl/blockchain/AionBlockchainImpl.java]: added a Jira task for this and enhancing the time estimates

**Pull request merged**